### PR TITLE
[User accounts] Initialize sites array

### PR DESCRIPTION
**Pull request by suzanne-lee**
In this PR I initialized the list_of_sites array in modules/user_accounts/php/NDB_Menu_Filter_user_accounts.class.inc because I was getting an error in the user accounts module (for some reason I had no site affiliations and so the foreach loop wasn't being run)

### CODE REVIEW

**Comment by AnyhowStep** [comment]
LGTM

File: garbage.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+djdjdjd
```
Comment (gluneau) [inline garbage.txt]: @suzanne-lee Is this file really needed?

**Comment by MounaSafiHarab** [comment]
@suzanne-lee 

I don't think your branch is up to date. This initialisation is already in the codebase!
Please confirm and if you agree, I think you can close this.
